### ISSUE TITLE
move to newer tifffile version 

### ISSUE THREAD

**Issue by burgerdev** [ilastik/lazyflow#203]
Is there a good reason to stay on tifffile-0.5? [Line 9ff](https://github.com/ilastik/lazyflow/blob/master/lazyflow/operators/ioOperators/opTiffReader.py#L9) in `OpTiffReader` don't work with tifffile-0.7, which has a [similar looking warning](https://github.com/blink1073/tifffile/blob/master/tifffile/tifffile.py#L234).

**Comment by stuarteberg** [comment]
I guess we've got no real reason to stay with 0.5, but I know of no real reason to update, either.  If you really want to upgrade, then be sure to update the [recipe](https://github.com/ilastik/ilastik-build-conda/tree/master/tifffile) in ilastik-build-conda and test it with the appropriate lazyflow changes.

**Comment by burgerdev** [comment]
From [official changelog](http://www.lfd.uci.edu/~gohlke/code/tifffile.py.html) (my italics):

2015.09.21
   * *Read RGB OME-TIFF.*
   * Warn about malformed OME-XML.

2015.09.16
   * *Detect some corrupted ImageJ metadata.*
   * Better axes labels for 'shaped' files.
   * Do not create TiffTags for default values.
   * Chroma subsampling is not supported.
   * Memory-map data in TiffPageSeries if possible (optional).

2015.08.17
   * Pass 1906 tests.
   * Write ImageJ hyperstacks (optional).
   * *Read and write LZMA compressed data.*
   * Specify datetime when saving (optional).
   * Save tiled and color-mapped images (optional).
   * Ignore void byte_counts and offsets if possible.
   * Ignore bogus image_depth tag created by ISS Vista software.
   * Decode floating point horizontal differencing (not tiled).
   * Save image data contiguously if possible.
   * Only read first IFD from ImageJ files if possible.
   * *Read ImageJ 'raw' format (files larger than 4 GB).*
   * TiffPageSeries class for pages with compatible shape and data type.
   * Try to read incomplete tiles.
   * Open file dialog if no filename is passed on command line.
   * Ignore errors when decoding OME-XML.
   * **Rename decoder functions (backwards incompatible)**

I guess there were some nice features added since `2014.08.24`, but the API has changed (last bullet). Not sure how much work it would be to adapt.

**Comment by stuarteberg** [comment]
OK, that's a convincing changelog.  I guess it would be nice to have those improvements.  The `OpExportTiff` and `OpMultipageTiffSequence` operators have simple unit tests, but alas, the `OpTiffReader` does not have a test at all.  Ideally, we should add a test using example OME-TIFF data:

http://www.openmicroscopy.org/site/support/ome-model/ome-tiff/data.html

Once a test is in place, it should be easy to see if upgrading `tifffile` is a simple drop-in replacement.

**Comment by stuarteberg** [comment]
Updated to tifffile `0.9.2` via bac6be66037784ce82a24a93b4d591af3cbd6c1c.
